base/src/com/google/idea/blaze/base/command/mod/BlazeModRunnerImpl.java

```diff
@@ -38,15 +38,32 @@
 
 public class BlazeModRunnerImpl extends BlazeModRunner {
 
+  private static final String DUMP_REPO_MAPPING = "dump_repo_mapping";
+  private static final String ROOT_WORKSPACE = "";
+
+  /**
+   * {@code bazel mod dump_repo_mapping} takes a canonical repository name and will dump a map
+   * from repoName -> canonicalName of all the external repositories available to that repository
+   * The name {@code ""} is special and considered to be <em>the main workspace</em> so in order to dump the main
+   * repository map we would invoke it like {@code bazel mod dump_repo_mapping ""}.
+   * <p />
+   * Additionally the flag {@code --enable_workspace} needs to be off for this to work. The flag is default
+   * off in bazel 8.0.0 but it is on between 7.1.0 and 8.0.0. So we need to also pass this along in
```

Comment:
is this flag supported by Bazel <7.1.0? If not, doesn't it break Bazel 6 and 5?

Reply:
This will not get called at all before 7.1.0 since the dump command itself didn't exist before.